### DISCUSSION TITLE
"Options" versus "settings"?

### DISCUSSION THREAD

**Discussion by SwuduSusuwu** [unifiedjs/unified#253] [Q&A] [answered]
https://github.com/unifiedjs/unified?tab=readme-ov-file#example-8 has
```
import {unified} from 'unified'

unified()
  // Plugin with options:
  .use(pluginA, {x: true, y: true})
  // Passing the same plugin again merges configuration (to `{x: true, y: false, z: true}`):
  .use(pluginA, {y: false, z: true})
  // Plugins:
  .use([pluginB, pluginC])
  // Two plugins, the second with options:
  .use([pluginD, [pluginE, {}]])
  // Preset with plugins and settings:
  .use({plugins: [pluginF, [pluginG, {}]], settings: {position: false}})
  // Settings only:
  .use({settings: {position: false}})
```
, but do not understand the differences (it looks as if both "options" and "settings" have the same syntax/purposes). Wish you explain this.

**Comment by ChristianMurphy** [comment, marked as answer]
Settings are for the parser and compiler https://github.com/unifiedjs/unified?tab=readme-ov-file#settings
Options apply to a transformer https://github.com/unifiedjs/unified?tab=readme-ov-file#processoruseplugin-options
